test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -21,76 +21,69 @@
-                    "The test cannot be run because SystemTray is not supported.",
-                    "Simply press PASS button."
-            };
-        } else {
-            String clickInstruction;
-            if (Platform.isOSX()) {
-                clickInstruction = "right";
-            } else {
-                clickInstruction = "left";
-            }
-            instructions = new String[]{
-                    "When the test starts, it adds the icon to the tray aread. If you",
-                    "  don't see a tray icon, please, make sure that the tray area",
-                    "  (also called Taskbar Status Area on MS Windows, Notification",
-                    "  Area on Gnome or System Tray on KDE) is visible.",
-                    "Double-click with " + clickInstruction + " button on the tray icon to trigger the",
-                    "  action event. Brief information about action events is printed",
-                    "  below. After each action event the tray icon is removed from",
-                    "  the tray and then added back in a second.",
-                    "The test performs some automatic checks when removing the icon. If",
-                    "  something is wrong the corresponding message is displayed below.",
-                    "  Repeat double-clicks several times. If no 'Test FAILED' messages",
-                    "  are printed, press PASS button else FAIL button."
-            };
+    public static void main(String[] args) throws Exception {
+        if (!SystemTray.isSupported()) {
+            throw new jtreg.SkippedException("The test cannot be run because " +
+                    "SystemTray is not supported.");
         }
-        Sysout.createDialogWithInstructions(instructions);
-    }
+        String clickInstruction = (Platform.isOSX()) ? "Right-click" : "Double-left click";
+
+        String instructions = "When the test starts, it adds the icon to the tray area. If you\n" +
+                       "  don't see a tray icon, please, make sure that the tray area\n" +
+                       "  (also called Taskbar Status Area on MS Windows, Notification\n" +
+                       "  Area on Gnome or System Tray on KDE) is visible.\n\n" +
+                        clickInstruction + " the button on the tray icon to trigger the\n" +
+                       "  action event. Brief information about action events is printed\n" +
+                       "  in the frame. After each action event the tray icon is removed from\n" +
+                       "  the tray and then added back in a second.\n\n" +
+                       "The test performs some automatic checks when removing the icon. If\n" +
+                       "  something is wrong the corresponding message is displayed below.\n" +
+                       "  Repeat clicks several times. If no 'Test FAILED' messages\n" +
+                       "  are printed, press PASS button else FAIL button.";
```

Comment:
It's not displayed below, it's to the right, isn't it?

Fail the test automatically if you can detect the failure. You may show a message to the tester about the failure before shutting down the test.

Reply:
This comment also implied amending the instructions to correspond to what the test does.

Reply:
the test doesn't actually do any automatic checks, so i've removed that line